From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ricardo Pardini <ricardo@pardini.net>
Date: Tue, 8 Aug 2023 12:29:37 +0200
Subject: kvim4/kvim4n: use different model names in DT

So we can tell them apart in userspace.
I wasted a month booting the wrong DT, since Khadas's upstream didn't differentiate them.
---
 arch/arm64/boot/dts/amlogic/kvim4.dts  | 2 +-
 arch/arm64/boot/dts/amlogic/kvim4n.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/kvim4.dts b/arch/arm64/boot/dts/amlogic/kvim4.dts
index d9e4ac48ee55..1599d39ebcfb 100644
--- a/arch/arm64/boot/dts/amlogic/kvim4.dts
+++ b/arch/arm64/boot/dts/amlogic/kvim4.dts
@@ -12,7 +12,7 @@
 #include "khadas-panel.dtsi"
 
 / {
-	model = "Khadas VIM4";
+	model = "Khadas VIM4 Classic (without NPU, A311D2 Revision B, T7 family)";
 	amlogic-dt-id = "t7_a311d2_vim4";
 	compatible = "amlogic, t7";
 	interrupt-parent = <&gic>;
diff --git a/arch/arm64/boot/dts/amlogic/kvim4n.dts b/arch/arm64/boot/dts/amlogic/kvim4n.dts
index 666ca5733331..8441b7f3dd6a 100644
--- a/arch/arm64/boot/dts/amlogic/kvim4n.dts
+++ b/arch/arm64/boot/dts/amlogic/kvim4n.dts
@@ -12,7 +12,7 @@
 #include "khadas-panel.dtsi"
 
 / {
-	model = "Khadas VIM4";
+	model = "Khadas VIM4 New (with NPU, A311D2 Revision C A311D2-N0D, T7C family)";
 	amlogic-dt-id = "t7c_a311d2_vim4";
 	compatible = "amlogic, t7c";
 	interrupt-parent = <&gic>;
-- 
Armbian

